pkgs/development/python-modules/biosppy/default.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  setuptools,
+  bidict,
+  h5py,
+  matplotlib,
+  numpy,
+  scikit-learn,
+  scipy,
+  shortuuid,
+  six,
+  joblib,
+  pywavelets,
+  mock,
+  opencv-python,
+}:
+
+buildPythonPackage rec {
+  pname = "biosppy";
+  version = "2.2.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "scientisst";
+    repo = "BioSPPy";
+    tag = "v${version}";
+    hash = "sha256-U0ZftAlRlazSO66raH74o/6eP1RpmuFoA6HJ+xmgKR8=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    opencv-python
+    bidict
+    h5py
+    matplotlib
+    numpy
+    scikit-learn
+    scipy
+    shortuuid
+    six
+    joblib
+    pywavelets
+    mock
+  ];
+
+  pythonImportsCheck = [
+    "biosppy"
```

Comment:
note (optional): you can add additional modules/packages to `pythonImportsCheck` e.g. `biosppy.signals`, etc. to increase your confidence in the package being correctly packaged (especially in the absence of tests).